.github/workflows/tests-pro-integration.yml

```diff
@@ -24,16 +24,21 @@ on:
         type: string
   pull_request:
     paths:
-      - ".github/workflows/tests-pro-integration.yml"
-      - "localstack/**"
-      - "tests/**"
-      - "setup.py"
-      - "pyproject.toml"
-      - "setup.cfg"
-      - "Dockerfile"
-      - "Dockerfile.rh"
-      - "docker-compose.yml"
-      - "bin/**"
+      - '**'
+      - '!.github/**'
+      - '.github/workflows/tests-pro-integration.yml'
+      - '!doc/**'
+      - '!scripts/**'
```

Comment:
The pro workflow uses a script in `scripts/`. While it does not usually get changed, it makes sense to include it. It's `./scripts/build_common_test_functions.sh`.

Reply:
Nice catch, thanks! I explicitly included this script in both triggers (`push` and `pull_request`).